### PR TITLE
update the license year automagically

### PR DESCRIPTION
**Pull request by rerpha**

### CODE REVIEW

File: utils/common_utils.py [hunk 1]
```diff
@@ -61,3 +61,9 @@ def run_command(command, working_dir):
         cmd = subprocess.Popen(command, cwd=working_dir, stdout=null_out, stderr=subprocess.STDOUT,
                                stdin=subprocess.PIPE)
     cmd.wait()
+
+def get_year() -> str:
+    """
+    Get the current year. 
+    """
+    return str(datetime.datetime.now().year)
```
Comment (JackEAllen) [inline utils/common_utils.py]: **Nitpick**: Could we add the return parameter to the docstring either as is or place the return statement in a variable and add the return parameter to the docstring?

Maybe something like this:
```python
def get_year() -> str:
    """
    Get the current year.
 
    Returns:
        current_year The current year formatted as a string
    """
    current_year = str(datetime.datetime.now().year)
    return current_year
```
Or
```python
def get_year() -> str:
    """
    Get the current year.
 
    Returns:
         str:  The current year formatted as a string
    """
    return str(datetime.datetime.now().year)
```

File: utils/support_utils.py [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-from utils.common_utils import run_command
+from utils.common_utils import run_command, get_year
 from utils.file_system_utils import append_to_file, mkdir, add_to_makefile_list, replace_in_file, copy_file
 from utils.command_line_utils import get_input
 from os import path, remove, linesep
 from shutil import copyfile
 import logging
 from utils.git_utils import RepoWrapper
-
+import datetime
```
Comment (JackEAllen) [inline utils/support_utils.py]: Is datetime import needed here?

**Review by JackEAllen — APPROVED** [review]
Looks good overall and works as expected! 🥇 
Just left 1 minor nitpick to improve the doctring inside `utils\common_utils.get_year()` and a quick question about the datetime import usage inside `utils\support_utils.py`

File: utils/support_utils.py (hunk 2 above)
Comment (rerpha) [reply utils/support_utils.py]: nope, good spot

**Comment by JackEAllen** [comment]
Thanks for making the changes, merging in now

**Pull request merged**